playbooks/tasks/portal-firewall-iptables-block-unblock.yml

```diff
@@ -20,6 +19,7 @@
     source: "{{ item }}"
     jump: DROP
   loop: "{{ private_vars.incoming_ips_ip_ranges_block }}"
+  when: private_vars.incoming_ips_ip_ranges_block is defined
```

Comment:
I would rather use empty list in private vars then skip the task if undefined.
If the portal operator misconfigures something he will never know this task is skipped.
When we want to skip this task and a couple of other long running tasks during portal setup rerun,
there is already a flag that can be set: `portal_setup_following_skip_on_rerun: True`